dynd/nd/__init__.py

```diff
@@ -1,5 +1,63 @@
 import sys
 
+import os
+if os.name == 'nt':
+    # Manually load dlls before loading the extension modules.
+    # This is handled via rpaths on Unix based systems.
+    import ctypes
+    import os.path
+    import dynd.ndt
+    ndtdir = os.path.join(os.path.dirname(os.path.abspath(dynd.ndt.__file__)), '..')
+    nddir = os.path.join(os.path.dirname(__file__), '..')
+    def load_dynd_dll(ndtdir=ndtdir, nddir=nddir):
+        try:
+            ctypes.cdll.LoadLibrary(os.path.join(ndtdir, 'libdyndt.dll'))
```

Comment:
I agree `libdyndt.dll` shouldn't be loaded here, because it `dynd.nd` requires `dynd.ndt` which is where it should be loaded.